frontend/src/metabase/query_builder/components/view/NativeQueryButton.jsx

```diff
@@ -84,6 +88,7 @@ export default class NativeQueryButton extends React.Component {
           <SqlIconButton iconSize={size} onClick={this.handleOpen} />
         </Tooltip>
         <Modal
+          style={{ padding: "1em" }}
```

Comment:
Add padding so the modal will have some spacing on the left and right on small screen devices.